pytorch_toolkit/object_detection/tests/export_tests_face_detection.py

```diff
@@ -29,29 +29,21 @@ def setUp(self):
     return ExportTestCase
 
 
-class FaceDetection0100TestCase(face_detection_test_case('face-detection-0100', True)):
-    """ Test case for face-detection-0100 model export. """
+class FaceDetection0100TestCase(face_detection_test_case('face-detection-0200', True)):
```

Comment:
Test cases use old model naming scheme. Is it by design?

Reply:
fixed